.travis.yml

```diff
@@ -2,7 +2,7 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode7.3
```

Comment:
That's against the baseline compiler, i.e. that's the patch we'd have to apply to Xcode--if we could. At least IIUC.